.github/workflows/release-container.yml

```diff
@@ -0,0 +1,44 @@
+name: Create Docker Image
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
```

Comment:
This will not always update to the latest nightly, since this action might be run before our publish step. Can we run this from the main release action using workflow calls? See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call

Reply:
I am not sure what your internal process/pipeline for release is, therefore it doesn't make much sense for me to poke around. Also I don't have more time to dedicate to this. I've gotten foundry installed in a docker container and this works for my team.

If you guys want to merge this code into some development branch and test it against your CI/CD pipeline, please feel free to do so. I think this can potentially make it easier for new users to quickly onboard to `foundry`.  At the very minimum, I would recommend maintaining an image that simply downloads the latest `forge` and `cast` binary.

Thank you